IFC4x3/PropertyConstants/a/ARCH_1vZI2DqO5BawEd_IF3tMVT.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ARCH_1vZI2DqO5BawEd_IF3tMVT" Name="ARCH" UniqueId="798d208d-d181-4b93-a3a7-f923c3dd67dd" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ARCH_1vZI2DqO5BawEd_IF3tMVT" Name="ARCH" UniqueId="798d208d-d181-4b93-a3a7-f923c3dd67dd">
```

Comment:
Could this constant be reused? Not sure about our general policy on reusing constants other than USERDEFINED and NOTDEFINED. Might as well resolve it "right here, right now!" :)

Reply:
well this is a propertyconstant the only other Arch is a "CONSTANT".....